### DISCUSSION TITLE
Text input - Space between Prefix and label

### DISCUSSION THREAD

**Discussion by andredelramo-hm** [Telefonica/mistica-design#1925] [Proposals]
### Goal

Actualmente el espacio entre el prefijo y el label del Text input es de 12px, visualmente me encaja que sea como la separación de un espacio entre caracteres y añadirlos en las specs.

https://www.figma.com/design/Lqf9jlyKYi0n9y2iiKk5Py/prefix?node-id=0-1&t=N8L0x6trtwczFlKP-1

### Context

-

### Impact

-

### Discussion Questions

-

### Additional considerations

-

**Comment by AnaMontes11** [comment]
Hello @andredelramo-hm thank you for your suggestion.

We understand that visually, the 12px spacing may seem wide, but we believe the separation between the prefix and the label in the Text Input should ensure a clear distinction between different groups of information.

This spacing is not only used for phone prefixes but also applies to other use cases such as:
- Currency symbols (€ 100)
- Units of measurement (kg, cm, ml)
- Web addresses or email addresses (www.example.com, user@email.com)

Proper spacing helps prevent confusion and improves readability, especially when handling different formatting conventions or numerical representations.
